src/components/banner/template.njk

```diff
@@ -1,5 +1,23 @@
+      <path d="M67.47,7.72a59.76,59.76,0,1,0,59.75,59.75A59.75,59.75,0,0,0,67.47,7.72m67.47,59.75A67.47,67.47,0,1,1,67.47,0a67.47,67.47,0,0,1,67.47,67.47" />
+    </svg>
+  <!--<![endif]-->
+  <!--[if IE 8]>
+    <img src="/assets/components/banner/images/hmrc_crest_18px.png">
```

Comment:
@tomshaw1 instead of a hardcoded src here we'd need to use something like an assets_url param like the header uses:
- param defined in the component options https://github.com/hmrc/hmrc-frontend/blob/fe15941b5e73b1d427b1594916f9440a0864b87f/src/components/header/header.yaml#L6-L9
- used in template https://github.com/hmrc/hmrc-frontend/blob/fe15941b5e73b1d427b1594916f9440a0864b87f/src/components/header/template.njk#L48

Reply:
Thanks Oscar. I've changed the path to reflect this

Reply:
@tomshaw1 on this one you've done exactly what I asked, but now I see it I think there's something we might want to adjust (apologies for not seeing this before!) At the moment we've got `assetsPath` on the header described as "The public path for the assets folder." but actually the default value is not that, it's the path to the images folder within the assets folder. We inherited this from govuk https://github.com/alphagov/govuk-frontend/blob/d86739ea1bd5a2e866209bba3863a5820224db78/src/govuk/components/header/template.njk#L36 but they are a bit inconsistent themselves https://github.com/alphagov/govuk-frontend/blob/d86739ea1bd5a2e866209bba3863a5820224db78/package/govuk/template.njk#L17 and I think in this case, really assetsPath or assetPath should follow the convention of the second example (template.njk) where it's just to the root /assets then everything that comes after that should be hard coded into the components template themselves. So here it would change to `default('/assets') }}/images/govuk-logotype-crown.png"`. Otherwise keeping all the values for this in sync will be quite challenging. For example if I change the assetPath for the template, then to adjust it I can't just pass the value for the template to the header/banner as well, I have to know what the additional suffixed path is and adjust the assetsPath I use for each component.

Reply:
@oscarduignan that makes total sense, I've made that change now too thanks